servicemodels/e2sm_rc/v1/e2sm_rc.proto

```diff
@@ -383,8 +383,8 @@ message EventTriggerUeeventInfoItem {
 // range of Integer from e2sm_rc.asn1:200
 // {RANParameter-ID}
 message RanparameterId {
-    // @inject_tag: aper:"valueLB:1,valueUB:4294967296,valueExt"
-    int64 value = 1 [(validate.v1.rules).int64 = {gte: 1, lte: 4294967296}, json_name="value"];
+    // @inject_tag: aper:"valueLB:1,valueUB:4294967295,valueExt"
+    int64 value = 1 [(validate.v1.rules).int64 = {gte: 1, lte: 4294967295}, json_name="value"];
 };
 
```

Comment:
I don't see here a change for 'RanparameterStructure' message - in the previous PR its only item was marked as optional (which corresponds to the specification). Can you please include this change here as well?